example/llm/flake.nix

```diff
@@ -13,13 +13,45 @@
       inputs.process-compose-flake.flakeModule
     ];
     perSystem = { self', pkgs, lib, ... }: {
-      process-compose."default" = {
+      process-compose."default" = pc: {
         imports = [
           inputs.services-flake.processComposeModules.default
         ];
-        services.ollama."ollama1" = {
-          enable = true;
-          models = [ "llama2-uncensored" ];
+        services = {
+          # Backend service to perform inference on LLM models
+          ollama."ollama1" = {
+            enable = true;
+            # The models are usually huge, downloading them in every project directory can lead to a lot of duplication
+            dataDir = "$HOME/.services-flake/ollama1";
+            models = [ "llama2-uncensored" ];
+          };
+          # Get ChatGPT like UI, but open-source, with Open WebUI
+          open-webui."open-webui1" = {
```

Comment:
How about renaming `open-webui1` to `instance1` ? I found it confusing as it is now... don't you?

Reply:
Yes, I could go with that. Once I add the `namespace` setting to the `open-webui` service that should indicate that the current process will be `instance1` under `open-webui` namespace.

Reply:
I don't know, I quite like it the way it is.  "ollama1-models" is more clear to me than "instance1-models" for instance.

Reply:
I was trying it out, it does make sense to keep it as it is, for readability. The line adding `depends_on` relation between open-webui and ollama process would look like:
```nix
{
  # Start the Open WebUI service after the Ollama service has finished initializing and loading the models
  settings.processes.instance1.depends_on.ollama1-models.condition = "process_completed_successfully";
}
```

vs

(Current)
```nix
{
  # Start the Open WebUI service after the Ollama service has finished initializing and loading the models
  settings.processes.open-webui1.depends_on.ollama1-models.condition = "process_completed_successfully";
}
```